data/presets/man_made/ceremonial_gate/paifang.json

```
@@ -0,0 +1,31 @@
+{
+    "icon": "roentgen-city_gate",
+    "fields": [
+        "{man_made/ceremonial_gate}"
+    ],
+    "moreFields": [
+        "{man_made/ceremonial_gate}"
+    ],
+    "geometry": [
+        "point",
+        "vertex",
+        "line",
+        "area"
+    ],
+    "terms": [
+        "chinese gate",
+        "paifong",
+        "pailau",
+        "pailou",
+        "shanmen"
+    ],
+    "tags": {
+        "man_made": "ceremonial_gate",
+        "ceremonial_gate": "paifang"
+    },
```

I wonder if we should do it like this. This should make it that the preset is selected based on only the `tags` and then the `man_made` is added. It would also trigger the "add missing tag" message in iD when only one tag is given.  But given that I messed this up before https://github.com/openstreetmap/id-tagging-schema/pulls we need to test this…

```suggestion
    "tags": {
        "ceremonial_gate": "paifang"
    },
    "addTags": {
        "man_made": "ceremonial_gate",
        "ceremonial_gate": "paifang"
    },
```